tox.ini

```diff
@@ -0,0 +1,94 @@
+# Global tox file
+
+# This file is used to invoke tox in individual charms
```

Comment:
[nit]: I don't think these two comments are necessary. I think it's clear based on the file's location within the repository that it is the root tox file.

Reply:
Removed them.